src/Microsoft.ML.Dnn/ImageClassificationTrainer.cs

```diff
@@ -363,7 +363,7 @@ public sealed class Options : TrainerInputBaseWithLabel
             /// Specifies the model architecture to be used in the case of image classification training using transfer learning.
             /// </summary>
             [Argument(ArgumentType.AtMostOnce, HelpText = "Model architecture to be used in transfer learning for image classification.", SortOrder = 15)]
-            public Architecture Arch = Architecture.InceptionV3;
+            public Architecture Arch = Architecture.ResnetV250;
```

Comment:
Why this change? This PR seems like an odd place to make this change.

Reply:
yep, I will describe in the PR that in addition it also does this this and this.

---
In reply to: [339649528](https://github.com/dotnet/machinelearning/pull/4395#discussion_r339649528) [](ancestors = 339649528)

Reply:
I didn't see a description of why this was changed.